### ISSUE TITLE
should logjam_agent send time in application timezone?

### ISSUE THREAD

**Issue by roccoblues** [skaes/logjam_agent#32]
During a migration I had the (unfortunate) situation that some logjam agents where sending requests and events in UTC time and some in CET. It looks like the data is stored with the timezone information in the logjam database but the view doesn't handle it correctly.

To fix that I hat to monkey patch logjam_agent to send all requests with the application timezone. I replace this two (https://github.com/skaes/logjam_agent/blob/master/lib/logjam_agent/rack/logger.rb#L29, https://github.com/skaes/logjam_agent/blob/master/lib/logjam_agent/rack/logger.rb#L51) `Time.now` occurences with `Time.current` which honors the application timezone.

For events I manually specified the start time:  `LogjamAgent.event(current_revision, started_at: Time.current.iso8601)`

**Comment by roccoblues** [comment]
See also: https://github.com/skaes/logjam_core/issues/67

**Comment by skaes** [comment]
Unfortunately this a somewhat complicated topic. Allowing arbitrary time zones in the sender will require the logjam-importer (and quite possibly also the devices) to convert incoming time stamps between time zones. Do you by any chance know a C-libary fro doing so?

**Comment by roccoblues** [comment]
I totally agree that it's complicated and no, unfortunately I don't know of such a C-lib.

Still wanted to add this topic here for future reference.
